README.md

```diff
@@ -18,48 +18,50 @@ At the end of the demo, you will have used SaltStack to automate:
 * Create a Compute Engine Load-balancer to distribute traffic over the 4 instances
 * Do a live test of the full configuration
 
-This is intended to be a fairly trival example. The video and repo show off
-the integration between SalStack and Google Compute Engine.  And, this can be
+This is intended to be a fairly trivial example. The video and repo show off
+the integration between SaltStack and Google Compute Engine.  And, this can be
 the foundational tools for building more real-world configurations.
 
 ## Google Cloud Platform Project
 
 1. You will need to create a Google Cloud Platform Project as a first step.
 Make sure you are logged in to your Google Account (gmail, Google+, etc) and
-point your browser to https://console.developers.google.com/. You should see a
+point your browser to https://console.cloud.google.com/projectselector/compute/instances. You should see a
 page asking you to create your first Project.
 
 1. When creating a Project, you will see a pop-up dialog box. You can specify
 custom names but the *Project ID* is globally unique across all Google Cloud
 Platform customers.
 
 1. It's OK to create a Project first, but you will need to set up billing
-before you can create any virtual machines with Compute Engine. Look for the
-*Billing* link in the left-hand navigation bar.
+before you can create any virtual machines with Compute Engine. Find the menu icon at the top left, 
+then look for the *Billing* link in the navigation bar.
 
 1. In order for `salt-cloud` to create Compute Engine instances, you'll need a
-[Service Account](https://developers.google.com/console/help/new/#serviceaccounts)
+[Service Account](https://cloud.google.com/compute/docs/access/service-accounts#serviceaccount). 
+It's recommended that you use the "Compute Engine default service account" for this demo.
 Make sure to download (or generate a new) P12 formatted private key file
-for your Service Account. Also, make sure to record the *Email address* that
-ends with `@developer.gserviceaccount.com` since this will be required in the
-Salt configuration files.
+for your Service Account as you will need to upload it to your Salt master later on. Also, make sure 
+to record the *Email address* that ends with `@developer.gserviceaccount.com` since 
```

Comment:
For the default account, it remains at developer.gserviceaccount.com.  For new accounts, it's project-id.iam.gserviceaccount.com.  Your asking this question made me realize that creating a new account his not only safer but allows us to have a little more control over the name, and since they had to create a new account previously, it's isn't any more work on their end.  So, I changed it.